consensus/mempool_test.go

```diff
@@ -254,5 +254,13 @@ func (app *CounterApplication) Commit() abci.ResponseCommit {
 
 func (app *CounterApplication) PrepareProposal(
 	req abci.RequestPrepareProposal) abci.ResponsePrepareProposal {
-	return abci.ResponsePrepareProposal{BlockData: req.BlockData}
+	dataHash := types.ToTxs(req.Txs).Hash()
+	squareSize := len(req.Txs)
```

Comment:
same question regarding the relationship between squareSize and req.Txs

Reply:
[nit] can these placeholders look identical to `PrepareProposal` in abci/types/application.go